docs/source/getting-started/tabular-data/group-by.ipynb

```diff
@@ -0,0 +1,928 @@
+       "  text-align: right;\n",
+       "  white-space: pre-wrap;\n",
+       "}\n",
+       "</style>\n",
+       "<small>shape: (2, 5)</small><table border=\"1\" class=\"dataframe\"><thead><tr><th>column</th><th>aggregate</th><th>distribution</th><th>scale</th><th>accuracy</th></tr><tr><td>str</td><td>str</td><td>str</td><td>f64</td><td>f64</td></tr></thead><tbody><tr><td>&quot;HWUSUAL&quot;</td><td>&quot;Sum&quot;</td><td>&quot;Float Laplace&quot;</td><td>5762.02402</td><td>17261.481317</td></tr><tr><td>&quot;HWUSUAL&quot;</td><td>&quot;Len&quot;</td><td>null</td><td>null</td><td>0.0</td></tr></tbody></table></div>"
+      ],
+      "text/plain": [
+       "shape: (2, 5)\n",
+       "┌─────────┬───────────┬───────────────┬────────────┬──────────────┐\n",
+       "│ column  ┆ aggregate ┆ distribution  ┆ scale      ┆ accuracy     │\n",
+       "│ ---     ┆ ---       ┆ ---           ┆ ---        ┆ ---          │\n",
+       "│ str     ┆ str       ┆ str           ┆ f64        ┆ f64          │\n",
+       "╞═════════╪═══════════╪═══════════════╪════════════╪══════════════╡\n",
+       "│ HWUSUAL ┆ Sum       ┆ Float Laplace ┆ 5762.02402 ┆ 17261.481317 │\n",
+       "│ HWUSUAL ┆ Len       ┆ null          ┆ null       ┆ 0.0          │\n",
+       "└─────────┴───────────┴───────────────┴────────────┴──────────────┘"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "\n",
+    "query_work_hours = (\n",
+    "    context.query()\n",
+    "    .group_by(\"SEX\")\n",
+    "    .agg(pl.col(\"HWUSUAL\").fill_null(0).dp.mean((0, 98)))\n",
+    ")\n",
+    "\n",
+    "description = query_work_hours.accuracy(alpha=.05)\n",
+    "description"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This time the query description breaks down into two separate statistics for the resulting \"HWUSUAL\" column.\n",
+    "The mean is computed by dividing separate sum and length estimates.\n",
+    "Since the partition length is marked as public information, the length is released in the clear, without any noise.\n",
+    "\n",
+    "Reading the table, the smallest suitable noise scale parameter for the sum is 5762. \n",
+    "This may seem large, but remember this noisy sum will be divided by the length, reducing the variance of the final estimate.\n",
+    "You can divide the sum accuracy estimate by the public partition sizes to get disaggregated accuracy estimates for each partition. \n",
+    "\n",
+    "Be mindful that accuracy estimates for sums and means do not take into account bias introduced from clipping.\n",
+    "However, in this example, since the legend for `HWUSUAL` limits work hours between 0 and 98, \n",
+    "the clipping introduced for differential privacy will not further increase bias.\n",
+    "\n",
+    "When the release is made, it is consistent with generally-known estimates of work hours for men and women."
```

Comment:
I don't think we define the codes here? Maybe add a string column? I'd leave off the "generally-known estimates" part, but whatever you think best.